### PR TITLE
#119 Bug in reloading after removing issue from home page removed

### PR DESCRIPTION
**Pull request by ashwini571**
Now one can delete issues on next pages also.

### CODE REVIEW

**Comment by jarvisdev** [comment]
Nice solution. How did you come up with this idea? :grinning:

**Comment by ashwini571** [comment]
Just read the jQuery Event Delegation.

**Comment by jarvisdev** [comment]
Okay, now I understand that why it was not working on the second page because every time we load some new content dynamically the event handlers we used for the previous content do not apply on new content because when we applied event listeners (i.e. when document is ready ) the new content was not there on web page. 
Here comes the use of event bubbling in JQuery. When an event happens the DOM tree is traversed from child to it parents, grandparents and so on and if any of them matches the second parameter (here the element with **delete class**) and executes the anonymous function. This is how delegated event is created.

File: Projects/templates/Projects/home.html [hunk 1]
```diff
@@ -198,8 +198,9 @@ <h4>
                     {% endif %}
                 });
 
-                $('.delete').click(function (e) {
+                $(document).on('click',".delete",function (e) {
```
Comment (dbads) [inline Projects/templates/Projects/home.html]: :+1:

**Pull request merged**